payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/SimpleDialogElementUI.kt

```diff
@@ -37,7 +40,14 @@ fun SimpleDialogElementUI(
                             onConfirmListener()
                         }
                     ) {
-                        Text(confirmText)
+                        Text(
+                            text = confirmText,
+                            color = if (destructive) {
+                                Color.Red
```

Comment:
We should use `MaterialTheme.colors.error` here.

Reply:
Is error the same as destructive though? Happy to change it to error if you think this is correct.